react/__tests__/SliderTrack.test.tsx

```diff
@@ -61,7 +62,7 @@ describe('Basic rendering', () => {
 
     // slidesPerPage === totalItems
     rerender(
-      <SliderTrack centerMode="disabled" usePagination totalItems={5} infinite>
+      <SliderTrack centerMode="disabled" centerModeSlidesGap={undefined} usePagination totalItems={5} infinite>
```

Comment:
Same comment here, you don't have to pass `undefined`

Reply:
Same as above.